### PR TITLE
feat:added weather and refactor code

### PR DESCRIPTION
**Pull request by tushargoyal812**
weather

### CODE REVIEW

File: chrome-extension/src/App.css [hunk 1]
```diff
@@ -114,4 +114,53 @@ body{
...
+  top: -8rem;
+  left: 60rem;
+  color: white;
+}
+
+.weather-modal{
+  height: 15rem;
+  width: 22rem;
+  background-color: white;
+  border-radius: 0.2rem;
```
Comment (deepak29-git) [inline chrome-extension/src/App.css:131]: use px for less than 10

File: chrome-extension/src/utils-func/get-weather.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+const API_KEY="0d75f5a05ca3b4cee287142fefdcc9fc"
+
+  export const getWeather=async(lat,lon,setWeather)=>{
+    try {
+      const response=await axios.get(`https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=${API_KEY}&units=metric`)
+      console.log(response);
```
Comment (deepak29-git) [inline chrome-extension/src/utils-func/get-weather.js:7]: remove console

**Review by deepak29-git — APPROVED** [review]
variable name is good

**Review by sharmayatendra — APPROVED** [review]
Nothing major caught my eyes
good to merge

File: chrome-extension/src/App.css (hunk 1 above)
Comment (tushargoyal812) [reply chrome-extension/src/App.css:131]: done

File: chrome-extension/src/utils-func/get-weather.js (hunk 2 above)
Comment (tushargoyal812) [reply chrome-extension/src/utils-func/get-weather.js:7]: ok

**Pull request merged**